core/src/main/java/hudson/util/DirScanner.java

```diff
@@ -136,5 +136,39 @@ public void scan(File dir, FileVisitor visitor) throws IOException {
         private static final long serialVersionUID = 1L;
     }
 
+    /**
+     * Scans by using Ant GLOB syntax.
+     * <p>An initial basename is prepended as with {@link Full} <strong>if the includes and excludes are blank</strong>.
+     * Otherwise there is no prepended path. So for example when scanning a directory {@code /tmp/dir} containing a file {@code file},
+     * the {@code relativePath} sent to the {@link FileVisitor} will be {@code dir/file} if {@code includes} is blank
```

Comment:
This entire comment seems to be copy&pasted without actually applying to this class (or does it?)